Cargo.toml

```diff
@@ -21,12 +21,13 @@ bytes = "1"
 hostname = "0.3"
 serde_json = "1"
 futures-channel = "0.3"
-futures-util = { version = "0.3", features = ["sink"] }
+futures-util = { version = "0.3", features = ["sink", "io"] }
```

Comment:
Can add a dedicated feature flag for these additional dependency features and the `tokio-rustls` import?

Reply:
Perhaps we can keep close to the `features` decisions made by `reqwest` and call the feature flag `rustls-tls`.

https://github.com/seanmonstar/reqwest/blob/master/Cargo.toml